packages/persist/lib/controllers/persist.controller.ts

```diff
@@ -57,6 +58,9 @@ class PersistController {
             },
             false
         );
+        const logCtx = getExistingOperationContext({ id: String(activityLogId) });
```

Comment:
it is not confusing when you remember both exist but it might be an good idea to simplify so there is always one function to use. It might make the signature too complex though (`getOperationContext` signature is already quite a mouthful)

Reply:
I have changed everything to inject the dependency, let me know 👍🏻